docs/get_started/getting_started_riscv_cmake.md

```diff
@@ -25,7 +25,7 @@ $ ./build_tools/riscv/riscv_bootstrap.sh
 ```
 
 * RISC-V toolchain is built from https://github.com/llvm/llvm-project
```

Comment:
Is it possible to share the llvm-project (and also riscv-gnu-toolchain) branch used to build the toolchain?

Reply:
OK. I'll add that information.

Reply:
Thanks for the additional information!